pyproject.toml

```diff
@@ -46,5 +47,12 @@ balanced_wrapping = false
 known_typing = "typing"
 sections = "FUTURE,STDLIB,TYPING,THIRDPARTY,FIRSTPARTY,LOCALFOLDER"
 
+[tool.ruff]
+target-version = "py38"
+line-length = 120
+
+[tool.ruff.format]
+quote-style = "single"
```

Comment:
I saw that we currently use a lot of `'foo'` style strings instead of the ruff default `"bar"` strings.